src/Microsoft.ServiceFabric.Services.Remoting/FabricTransport/Runtime/FabricTransportRemotingListenerSettings.cs

```diff
@@ -32,6 +34,8 @@ public FabricTransportRemotingListenerSettings()
             this.headerBufferSize = Constants.DefaultHeaderBufferSize;
             this.headerMaxBufferCount = Constants.DefaultHeaderMaxBufferCount;
             this.useWrappedMessage = false;
+            this.remotingExceptionDepth = DefaultRemotingExceptionDepth;
+            this.ExceptionSerializationTechnique = ExceptionSerialization.BinaryFormatter;

```

Comment:
Are we keeping the default serializer as binary formatter?

Reply:
Thats right... We don't want to break customer apps with out notice, hence retaining the existing behavior